src/common/meta/src/ddl/create_flow.rs

```diff
@@ -206,21 +206,21 @@ impl From<&CreateFlowTaskData> for CreateRequest {
             expire_when: value.task.expire_when.clone(),
             comment: value.task.comment.clone(),
             sql: value.task.sql.clone(),
-            task_options: value.task.options.clone(),
+            flow_options: value.task.flow_options.clone(),
         }
     }
 }
 
-impl From<&CreateFlowTaskData> for FlowTaskInfoValue {
+impl From<&CreateFlowTaskData> for FlowInfoValue {
     fn from(value: &CreateFlowTaskData) -> Self {
         let CreateFlowTask {
             catalog_name,
-            task_name,
+            flow_name: task_name,
```

Comment:
CreateFlowTaskData still uses task_name
```suggestion
            flow_name,
```